<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三维井筒模型与风险井段识别</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>数据输入</string>
    </property>
    <addaction name="well_numbers"/>
    <addaction name="load_wellprofile"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>绘图</string>
    </property>
    <addaction name="plot_well_profile"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="load_wellprofile">
   <property name="text">
    <string>井斜参数</string>
   </property>
  </action>
  <action name="plot_well_profile">
   <property name="text">
    <string>三维井眼轨迹图</string>
   </property>
  </action>
  <action name="well_numbers">
   <property name="text">
    <string>井数</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
